src/gui/properties/speedplotview.h

```diff
@@ -29,9 +29,12 @@
 #ifndef SPEEDPLOTVIEW_H
 #define SPEEDPLOTVIEW_H
 
+#ifndef Q_MOC_RUN
```

Comment:
This bug fixed in Qt 5.+ but not in Qt 4.* https://bugreports.qt.io/browse/QTBUG-22829